tests/tests/coins.rs

```diff
@@ -613,7 +613,7 @@ mod all_coins {
             .await
             .unwrap();
         assert_eq!(coins_per_asset.len(), 2);
-        assert!(coins_per_asset[0].len() >= 1);
+        assert!(coins_per_asset[0].is_empty());
```

Comment:
is this a functional change in your pr?

Reply:
Same as https://github.com/FuelLabs/fuel-core/pull/2786#discussion_r1979787265